catt/controllers.py

```diff
@@ -208,12 +211,12 @@ def __init__(self, state_path):
     def get_data(self, name):
         try:
             data = self._read_store()
-            save_data = data.get(name)
-            if save_data and set(save_data.keys()) != set(["controller", "data"]):
+            if set(next(iter(data.values())).keys()) != set(["controller", "data"]):
```

Comment:
This is obscure enough that a comment is warranted, I think. My problem with Python in general is that I never know the type, so I can't reason about things. For example, I know you're looking for the first truthy value here, but I have no idea reading this line what type `data` is. Maybe a comment with what `data` looks like and a comment saying "Get the first truthy element" would clarify things more.

Reply:
LOL, yeah this looks pretty weird. The problem I was trying to solve is that keys in the root level are cc names, so I can't verify those, and so I'm just trying to validate the data associated with whatever cc name. Maybe I'm misunderstanding you, what do you mean by "truthy"?

Reply:
Oh sorry, I misunderstood the code. I'm confused as to why you do `next(iter(data.values()))` rather than just `data.values()[0]`?

Reply:
```data.values``` does not do indexing, as far as I can tell.

Reply:
Is `data` not just a dictionary? It's returned by `json.read()`, so it should be, no?

Reply:
Yep, exactly:
```'dict_values' object does not support indexing```

Reply:
Oh bah, I forgot that's a view object in py3... And it doesn't even support iteration, how inconvenient. `list(data.views())[0]` might be more readable, but not by much. Alright, looks good to me then!